portfolio.html

```diff
@@ -0,0 +1,50 @@
O96LqbWi5nkqwdHYxrIWGLIWQqLtg4j8Mjimsg/Yj302l6fHCpcIGPhsVEhyGSyqjD5VDEBu1EWDV1rz7b11RoVe4i+XarI4DC+VsUa+g9tVUySv+3I59TbH9+qkOOZU5Ggujumb8RbhJivCvkVXjjmwLPseLPprs3WJSqKrMgWNY2xYjLEVZquK9O2gwvhyASJYUgsl0SO9WO1jix769VtOl7bx0jpXhSMSM7WpZJZAVY+ZPkiZbJNKQ5o8Ah5bFxP3Qbi0K89JtppFiamfMFIwOWIjpeAOcQTQ+oPtopdikcSzOrMYpvC3ML2tXkVoqQwBCsDdEMvsdGPu1jiilU2ngT7RgpFq7GZlJFjysHDX64sO40LvurDcB1CN4m43CM6KfRVKKqsR8zM7HtxQ76JJJXQFq3Qwm7kDj/usO4VJHzXyo7kJlRsEqOfUc3R006p1rb+DNHmDJJFHGViH5KsqoLFGiFKkXTcVj3JPm9/1KWeScBSBLKZHjUWbF9yBkQtnvxdmtY7mKIlvCaQhUuyvc3X/CKN8/bSloJGKe9/PTf0QNX6dgM9Y6e6cy9bRDPHBCDt+THllkoJjJ8xNhGZAa79uR20t3kssm48WViviSK7uh+XPswN+38td3a/1qggflxlPqgAPH8/0Ogt1NnmQtjyi6+WuK44F6nTOK7bZc7hp9z5KLa76sbvTI283I6LaJGbdbMcximvkYtwrj2rn+Ghd+FxVXNvHlFx6gco329P3auY5Zo3lzBoxxuo4JHZGPoRYr3utFdK6fUk1KCFWRYi4GLSLXFnylgLIGiQW+Jxv38npyu9OkXPEm+ff3+25AbbbSbjyxRsxtLI7Kfl5+51t1DocqosrtlmX5AJGS8VY/aNe2nEvXYYDIUqRpkjBwJTw8UAPIHzZC+NIt/1mWdzZCZNka7k1j5j6mtK3HNhDf4HfRacFOm3j38LF9wqEJgOKyb15HP89Yx7oKKRfWxf2o/w1n4fmF/TVzOq1ivPv9dEwMlLEbRcwsfBPrxrsmIFDk66oZ+SaGt5tuipz83vo31RvmUIcjzXGjfwsSpbsc7+Uawk3ZIxXtoZu/Olhx4clYYW5CUX+Ii/8P8AjqaC1NHAOyuxlMdpsSJY1lQhWaueL1n4+LMQijngego+l8+mitzvwWXE+USZ0FoDketkk6E3wXNsWDAkni/U/XVYkQVNwBssHeyT7m9cvV0iJBIHC1f68DW+224KSO1+UAAe7MaH6AAn9NEcF0JjuJQIdoCwtGckKVZlUsrLxdWSTwf11TczS7vcTSRp5pCzlE449R6X6cep9NBNEEVWyIc0ygDsL4N3weLArTQzXuNrOKDvg78cZCQoWoV82IY1Xc6K5Ktrs5JLMaM1Ak0OwHJ016f0kB4mm/q2kRHVbyBYZAG6HI9QTX341hv9wqq8KZLjOxAuxWOB597H7jqm96k85XJQXGIDDIsSOBQyIF8cKBZ1ncaz7CwvfUccx5iITwxud+8skb0zbRHzYnyOQ2bDhCjAE8AcOP30NZ/0nVLZMfghcVFUxiK32HIY8n1+vGgFDSvTsxPNlrJFXxXf9NXjiXGyG4cK1muDlfpYPH19dE0hPjJNojdPfRIakCAFXebnNsiKNKCP7qhSf1q/11osBaRY5WK/KATbUCBjXPaiK5rnRaSRu7wpQSSsCRVOLKE/eyh/vduNDLMCsdkLJG4UZXyt2LPoUNj7Ee2nnw2t7Wt91EkkTqs5RGqV5zLlRP7AAJH8eP3HTXp8PgKzOqLPHMuSynF0VQHBRCQS5bjgEih2BJ0l3TAyOwFoXYi7oi7r0PY/fTODZS7yR5XbEu3DFTizENQu+FAWr5rgc66YbLjx/jkmb4RJR/XPiZizLC4dWUqzlTyM5GUBG4pVcKLWxjxWkEaveADZNQxANm6IFdzfH8NM+tdKhiwAfA26uTbWVCEEADgtl27VjzrfpPXFiCTG2kCGB47rMCsGJ9MVsBhyCiaVhaG4qbc+xPklqVC8A5rbovw+ZQGnJEdBVxdbBJrkc1RI8ponIdtbl/ywSFCPsTmFAVSySMqNQFZZBee/fSw9ankURxeRY4yMVP7IChibPsoPlrmzXJ1lIkkixIZFEfhApQoUWorQHLZ3ZN886NwSXmPZLTqfTkvNtyI3XWoyZXSMJkYVVPTyUxJN8klQNAed1ETuFQSFiPZmHzH91a2kZApqlSRLHsssfcfYm/8AzDS2feZZnm2NgfbShoDQ1ggCL5kZDXXPyTOqvquLhacz18s58l2SNR37YHkepvVPGBcBF4sfc62g6dJIoJAVBdM3/Lv30QqRpTstEYcDtz3I+orRtrdUbROZQabKRmAKkDsfpWtt/tYkKkMewJB7nVNxv5HxRf2BQK9zXqdZy7fkGR+Tyf8Ar310wbq0NiAo27JGKL2vn1/9tYBb5c60lnUeVBQ9/U65DtmcZHhR3OumBuXa71G3IAxRa+vrrn4NgMn8o+vc6ss6ofKL9r9/fWO43DSG2NnQvNrBcY1WuUXsdd0Jqa7BxPmungF3XdTU1YKac/Dv7f8Awf4tTqXbc/71P5Pqams1P893L/5V3flDn8rLedo/9wP8R0btv9Z2X9yL/EdTU1pGSiM0l3Hzt/eP8zrXb/NH/eH+LXNTQOSmnkf9fvf7sv8AiOlMf9S/9+L+UmpqazjN3+z1Cg3Py9kJrg1NTWpWOSawf6sn/wBWP8I16iX5m/3sX/7U+pqa8zac+p9Qo1sikfxj/wDA/wB2P8Ka88NTU1s2b8sdfUqlP8sdU1+H/wCt/wCB/wDA2toP9Xg+7/8A5ItTU0do15j/AJLPtGfl/wAklk7f8R1yL5k+4/nqamrFbW5DqvY7z5pf7n+Z14yTsfuP5amprNSyWisi+j/N+h1Xqf7P66mpo/rQ/wC2gl9fto1v9XH97U1NF2nMIM15IDU1NTTqa5qamprly//Z"   alt="coloured code" width="300"/>
```

Comment:
Oh my.. this image url is really long. Perhaps saving the photo and adding it to your project in an images folder would clean up your code.

Reply:
Great suggestion, Audrey, I updated my code!